build_tools/setup_helpers/extension.py

```diff
@@ -94,6 +99,8 @@ def _get_extra_objects():
         ]
         for lib in libs:
             objs.append(str(_TP_INSTALL_DIR / 'lib' / lib))
+    if _BUILD_TRANSDUCER:
+        objs.append(str(_TP_BASE_DIR / 'build' / 'transducer' / 'libwarprnnt.a'))
```

Comment:
This mean it is linking to the library in build directory, not the one installed with `install` command.

Reply:
If they were not, I'd expect the nightlies to fail on my computer, right?
```
base ❯ conda create -n pytorch-nightly -c pytorch-nightly pytorch torchaudio            
base ❯ conda activate pytorch-nightly
pytorch-nightly ❯ python -c "from torchaudio.prototype.transducer import RNNTLoss; import torchaudio; print(torchaudio.__version__)"
0.8.0a0+361a146
pytorch-nightly ❯ 
```
Or did you find a different case that failed?

Reply:
It's not about nightly but about how CMake organize the intermediate files and directories inside of build directory. (nighty build has the library data statically linked. so, as long as build succeed, it is not an issue.) This is more about maintainability.

The file being pointed here is an intermediate product that is waiting to be installed. I do not think there is any guarantee that CMake will keep it this way, or CMake in different system could put it differently. Also this way, users do not have control over where the resulting binary is found.
(this way, the INSTALL command in CMakeLists.txt is not used)

So, more future proof way to do this is to perform `install` step and install the resulting binary in pre-determined directory. I cleaned this up in #1159